extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
+// By: BOP <https://scratch.mit.edu/users/CEFSharp/>
+
+(function (Scratch) {
+  "use strict";
+
+  let chat = null;
+  let token = null;
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
+        name: "Twitch",
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
```

Comment:
Please make this a dataURI

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/07f90cc5-378a-46b0-9d32-7d2b9475c15d)
![image](https://github.com/TurboWarp/extensions/assets/140775902/0d2fd97a-54c7-4c43-88d9-3cb87e48c133)